tests/test_series.py

```diff
@@ -998,6 +998,11 @@ def test_types_getitem() -> None:
     value: int = s2[0]
     s3: pd.Series = s[:2]
 
+def test_types_getitem_by_timestamp() -> None:
+    index = pd.date_range("2018-01-01", periods=2, freq="D")
+    series = pd.Series(range(2), index=index)
+    value: int = series[index[-1]]
```

Comment:
Can you replace this with `check(assert_type(series[index[-1]], int), int)`

Reply:
The type check of the returned value was a nice to have. Should I just replace it with `series[index[-1]]` since it is not the subject of the test?

Reply:
With new tests, we are always using `check` and `assert_type`, so I'm suggesting to replace `value: int = series[index[-1]]` with `check(assert_type(series[index[-1]], int), int)`